pkgs/development/python-modules/pyvex/default.nix

```diff
@@ -28,7 +28,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace pyvex_c/Makefile \
-      --replace "CC=gcc" "CC=${stdenv.cc.targetPrefix}cc"
+      --replace "CC?=gcc" "CC=${stdenv.cc.targetPrefix}cc"
```

Comment:
We could also set the CC env now.

Reply:
Update to use env variable now. Thanks for the feedback.